lib/lte_link_control/Kconfig

```diff
@@ -228,7 +228,7 @@ config LTE_MODE_PREFERENCE_NBIOT_PLMN_PRIO
 
 endchoice
 
-config LTE_MODE_PREFERENCE
```

Comment:
Please mention in the commit message and PR description that this has been renamed and taken into use for another purpose.
And changelog entry for the same story.